scripts/west_commands/tests/test_imports.py

```diff
@@ -23,6 +23,7 @@ def test_runner_imports():
                     'hifive1',
                     'intel_s1000',
                     'jlink',
+                    'mdb',
```

Comment:
Just making a quick response here before I have time to review the Python in more detail, but I would really like to see more test cases here for maintainability, especially since the underlying tool is proprietary. Even just some basic sanity checking that the `self.check_call()` invocations happen in the correct order for simple default runs would be great.

Reply:
(existing runners without test cases are grandfathered in because the test framework is newer than the runners, but it would be nice not to make this issue worse)

Reply:
I am not a python expert, but i will try to prepare the unit test of mdb runner